src/quo/components/overlay/view.cljs

```diff
@@ -0,0 +1,26 @@
+(ns quo.components.overlay.view
+  (:require
+    [quo.components.overlay.style :as style]
+    [quo.foundations.colors :as colors]
+    [react-native.blur :as blur]
+    [react-native.core :as rn]))
+
+(defn view
+  [{:keys [type]} child]
+  (let [background-color (case type
+                           :shell  colors/neutral-80-opa-80-blur
+                           :drawer colors/neutral-100-opa-70-blur)]
```

Comment:
It's risky to not have a fallback when using `(case type ...)` because it's easy to get in a situation where a new `type` is added somewhere and then no match is found and an exception is thrown. This would be seen currently as a whitescreen to the user. So generally in status-mobile UI code, almost always handle *match not found* in the `case` macro. Here returning `nil` is probably fine.

Source code: https://github.com/clojure/clojurescript/blob/8eba755046ff3ae915ed3378574486840621a491/src/main/clojure/cljs/core.cljc#L2386